### DISCUSSION TITLE
PaddleX/latest/pipeline_usage/tutorials/time_series_pipelines/time_series_forecasting

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleX#2666] [General]
# PaddleX/latest/pipeline_usage/tutorials/time_series_pipelines/time_series_forecasting

https://paddlepaddle.github.io/PaddleX/latest/pipeline_usage/tutorials/time_series_pipelines/time_series_forecasting.html

**Comment by StatML** [comment]
预训练模型下载链接全部失效，望尽快解决，谢谢
